geometry_msgs/Point target_point
---
geometry_msgs/PoseStamped goal_pose
